crates/vm/levm/src/opcode_handlers/arithmetic.rs

```diff
@@ -220,45 +220,36 @@ impl VM {
         let byte_size = current_call_frame.stack.pop()?;
```

Comment:
I know this is not from this PR but I'd change the `byte_size` name for a more accurate one, like `byte_size_minus_one` or something like that. Just to avoid misunderstandings, because someone may think that `byte_size` represents the size in bytes of the value sent as input when it is actually `byte size - 1`.

Reply:
[Done! ](https://github.com/lambdaclass/ethrex/pull/1408/commits/c7bed6c3a2f7385e833a83a20038d7a7e4fe34ec)